compiler/damlc/daml-stdlib-src/DA/Internal/Date.daml

```diff
@@ -10,12 +10,12 @@ import DA.Internal.Prelude
 import DA.Internal.LF
 import GHC.Types (primitive)
 
--- | Convert from number of days since epoch (i.e. the number of days since
+-- | HIDE Convert from number of days since epoch (i.e. the number of days since
```

Comment:
If we expose `toEnum` and `fromEnum`, maybe we should just expose this as well?

Reply:
I think that's fair. done.